damus/Views/SearchResultsView.swift

```diff
@@ -88,8 +118,11 @@ struct InnerSearchResults: View {
             case .naddr(let naddr):
                 SearchingEventView(state: damus_state, search_type: .naddr(naddr))
             case .multi(let multi):
-                VStack {
-                    HashtagSearch(multi.hashtag)
+                VStack(alignment: .leading) {
+                    HStack {
+                        HashtagSearch(multi.hashtag)
+                        TextSearch(multi.text)
+                    }
```

Comment:
I feel like we should add more spacing between these buttons and the profile search results below it.

Right now the spacing between the button and the top looks much larger than the button and the profile results below it, making it look a bit asymmetrical.
<img width="1027" alt="Screenshot 2024-09-27 at 11 27 11" src="https://github.com/user-attachments/assets/d7c3d731-f70e-46c2-b714-59353c901705">